src/pages/FeaturePreviews.js

```diff
@@ -0,0 +1,90 @@
+import * as Style from 'src/libs/style'
+import * as Utils from 'src/libs/utils'
+
+
+export const FeaturePreviews = () => {
+  const { featurePreviews, loading, error } = useAvailableFeaturePreviews()
+
+  // The source of truth for whether or not a feature preview is enabled is `isFeaturePreviewEnabled`, which reads
+  // from local preferences. However, changing a local preference won't re-render this component and its checkboxes.
+  // Thus, we also need to store this in component state and keep in sync with local preferences.
+  const getFeaturePreviewState = useCallback(() => {
+    return _.flow(_.map(({ id }) => [id, isFeaturePreviewEnabled(id)]), _.fromPairs)(featurePreviews)
+  }, [featurePreviews])
+
+  // Map of feature preview ID => enabled boolean
+  const [featurePreviewState, setFeaturePreviewState] = useState(getFeaturePreviewState)
+  useEffect(() => { setFeaturePreviewState(getFeaturePreviewState()) }, [getFeaturePreviewState])
+
+  return Utils.cond(
+    [loading, () => spinnerOverlay],
+    [error, () => p({ style: { margin: 0 } }, ['Unable to load feature previews.'])],
+    [_.size(featurePreviews) === 0, () => p({ style: { margin: 0 } }, ['No feature previews available at this time.'])],
+    () => h(Fragment, [
+      p(['These features are proof-of-concept and may change without notice.']),
+      h(SimpleFlexTable, {
+        'aria-label': 'Features',
+        rowCount: featurePreviews.length,
+        columns: [
+          {
+            size: { basis: 55, grow: 0 },
+            field: 'enabled',
+            headerRenderer: () => span({ className: 'sr-only' }, ['Enabled']),
+            cellRenderer: ({ rowIndex }) => {
+              const { id, title } = featurePreviews[rowIndex]
+              return h(Checkbox, {
+                'aria-label': `Enable ${title}`,
+                checked: featurePreviewState[id],
+                onChange: checked => {
+                  toggleFeaturePreview(id, checked)
+                  setFeaturePreviewState(_.set(id, checked))
+                }
+              })
+            }
+          },
+          {
+            size: { basis: 250 },
+            field: 'description',
+            headerRenderer: () => 'Description',
+            cellRenderer: ({ rowIndex }) => {
+              const { title, description } = featurePreviews[rowIndex]
+              return div([
+                p({ style: { fontWeight: 600, margin: '0.5rem 0 0.5rem' } }, [title]),
+                p({ style: { margin: '0.5rem 0' } }, [description])
+              ])
+            }
+          }
+        ]
+      })
+    ])
+  )
+}
+
+const FeaturePreviewsPage = () => {
+  return h(FooterWrapper, [
+    h(TopBar, { title: 'Feature Preview' }),
+    h(PageBox, { role: 'main' }, [
+      h2({ style: { ...Style.elements.sectionHeader, textTransform: 'uppercase' } }, [
+        'Feature Preview'
+      ]),
+      h(FeaturePreviews)
+    ])
+  ])
+}
```

Comment:
[Nit] The following reads a bit easier to me FWIW  🤷🏼 
```suggestion
const FeaturePreviewsPage = () => h(FooterWrapper, [
  h(TopBar, { title: 'Feature Preview' }),
  h(PageBox, { role: 'main' }, [
    h2({ style: { ...Style.elements.sectionHeader, textTransform: 'uppercase' } }, [
      'Feature Preview'
    ]),
    h(FeaturePreviews)
  ])
])
```

Reply:
I dislike this style because it leads to massive diffs and messy history if you end up adding anything besides a return to the function. But I guess this one is unlikely to change.

Reply:
I can see that. I think filtering out white space changes (by appending `?w=1` to the GitHub URL) would take care of the additional diff's but even that could be inconvenient.